test/tests/rackhd20/test_rackhd20_api_config.py

```diff
@@ -44,14 +43,19 @@ def test_api_20_config_httpendpoints(self):
 
     def test_api_20_config_patch(self):
         api_data_save = fit_common.rackhdapi('/api/2.0/config')['json']
-        data_payload = {"CIDRNet": "127.0.0.1/22"}
+        if ("logColorEnable" in api_data_save and api_data_save['logColorEnable'] == True):
+            data_payload = {"logColorEnable": False}
+        else:
+            data_payload = {"logColorEnable": True}
         api_data = fit_common.rackhdapi("/api/2.0/config", action="patch", payload=data_payload)
         self.assertEqual(api_data['status'], 200, "Was expecting code 200. Got " + str(api_data['status']))
         for item in api_data['json']:
-            if fit_common.VERBOSITY >= 2:
-                print "Checking:", item
```

Comment:
I think its just noise anyways. This is a very simple test, the less noise the better.